packages/flutter/lib/rendering.dart

```diff
@@ -23,6 +23,7 @@
 library rendering;
 
 export 'package:flutter/foundation.dart' show
+  AbstractNode,
```

Comment:
i think you can skip this one, we don't really view AbstractNode as a part of the rendering API

Reply:
done